rakefile.rb

```diff
@@ -28,7 +28,7 @@
 
 def copy_to_app(app_target_relative_path)
   app_target_path = File.join(solution_dir, app_target_relative_path)
-  source_dir = File.join(src_dir, 'OSPSuite.UI', 'bin', 'Debug')
+  source_dir = File.join(tests_dir, 'OSPSuite.Starter', 'bin', 'Debug')

```

Comment:
copying form starter instead of UI just to be sure that we have every single DLL